routes/api.php

```diff
@@ -14,7 +14,7 @@
 */
 
 Route::get('links', 'Api\LinkController@index');
-Route::post('links', 'Api\LinkController@store');
+Route::middleware('auth:api')->post('links', 'Api\LinkController@store');
 
```

Comment:
you can just move `Route::post('links', 'Api\LinkController@store');` into the group below so you have 

```
Route::middleware('auth:api')->group(function () {
    Route::post('rooms', 'Api\RoomController@store');
});
```